### ISSUE TITLE
WCAG/ACCNAME discrepancy with `<img alt="" title="foo">

### ISSUE THREAD

**Issue by cookiecrook** [w3c/aria#602]
The following text in ACCNAME:
> "Otherwise, use the value as specified by a host language attribute." 

Seems to literally means the value even if it's empty, which does not leave the exception covered by WCAG Success Criteria H67: https://www.w3.org/TR/WCAG20-TECHS/H67.html
> For each image that should be ignored: Check that title attribute is either absent or empty. Check that alt attribute is present and empty. ...

Furthermore, both Blink and Gecko allow this behavior. WebKit currently matches the HTML-AAM and ACCNAME specs, but will likely change to match the other browser's behavior. https://webkit.org/b/173870

Test case 7 here:
https://bugs.webkit.org/attachment.cgi?id=314203

**Comment by cookiecrook** [comment]
Similar issue in HTML-AAM https://github.com/w3c/html-aam/issues/99

**Comment by jnurthen** [comment]
This issue was moved to w3c/accname#27
